src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -12649,7 +12653,8 @@ void VRFTerminalUnitEquipment::ControlVRF_FluidTCtrl(EnergyPlusData &state,
 
     // set supplemental heating coil calculation if the condition requires
     if (this->SuppHeatingCoilPresent) {
-        if ((QZnReq > DataHVACGlobals::SmallLoad && QZnReq > FullOutput) ||
+        if (((QZnReq > DataHVACGlobals::SmallLoad && QZnReq > FullOutput) ||
+             (((QZnReq - NoCompOutput) > DataHVACGlobals::SmallLoad) && QZnReq <= 0.0)) ||
             (this->isSetPointControlled && this->suppTempSetPoint > state.dataLoopNodes->Node(this->SuppHeatCoilAirInletNode).Temp)) {
```

Comment:
Shortcuts like `auto &thisSuppHeatCoilAirInletNode`, `auto &thisVRFTerm`, and `auto &thisVRFCond` could help make this function more readable.

Reply:
Done.